docs/cvl/genericrules.md

```diff
@@ -0,0 +1,18 @@
+Builtin Rules
```

Comment:
There is already a file for builtin rules called builtin.md

Reply:
moved there